modules/ztp-about-ztp-and-single-node-openshift-clusters-factory.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
```

Comment:
Per Jupiter guidelines:

> Each .adoc file must contain a content type attribute in the metadata: :_content-type: <TYPE>

Reply:
My bad, I initially started with this as part of content, but removed. I should have deleted these files and updated the PR accordingly, sorry about that.